### PR TITLE
run travis tests with Ruby 2.3

### PR DESCRIPTION
**Pull request by mmoll**
:green_heart: tests with 2.3:
https://travis-ci.org/mmoll/puppet-dhcp/builds/125962904
https://travis-ci.org/mmoll/puppet-dns/builds/125963413
https://travis-ci.org/mmoll/puppet-foreman/builds/125963552
https://travis-ci.org/mmoll/puppet-foreman_proxy/builds/125963734
https://travis-ci.org/mmoll/puppet-git/builds/125963910
https://travis-ci.org/mmoll/puppet-puppet/builds/125964291
https://travis-ci.org/mmoll/puppet-tftp/builds/125964901

### CODE REVIEW

**Comment by brandonweeks** [comment]
2.3.1? Otherwise 👍

File: moduleroot/.travis.yml [hunk 1]
```diff
@@ -37,5 +37,8 @@ matrix:
     # Only Puppet 4.x supports Ruby 2.2. Also limit the OS set we test Ruby 2.2 with.
     - rvm: 2.2.3
       env: PUPPET_VERSION=4.0 ONLY_OS="debian-8-x86_64,centos-7-x86_64,ubuntu-16-x86_64,freebsd-10-amd64,windows-2012 R2-x64"
+    # Only Puppet >= 4.4 supports Ruby 2.3. Also limit the OS set we test Ruby 2.3 with.
+    - rvm: 2.3.0
+      env: PUPPET_VERSION=4.0 ONLY_OS="debian-8-x86_64,centos-7-x86_64,ubuntu-16-x86_64,freebsd-10-amd64,windows-2012 R2-x64"
```
Comment (domcleal) [inline moduleroot/.travis.yml]: Should we remove some OSes here, e.g. debian-8 and centos-7?

Comment (mmoll) [inline moduleroot/.travis.yml]: I'd like to run that test with every major OS. As newer Rubies are faster, this is also quite cheap.

**Comment by mmoll** [comment]
@brandonweeks At least now there's only 2.3.0 available for travis as binary.

File: moduleroot/.travis.yml (hunk 1 above)
Comment (domcleal) [inline moduleroot/.travis.yml]: Okay.

2) Nitpick: PUPPET_VERSION here should be 4.4 to match the comment above, this will require `~> 4.4`

**Comment by domcleal** [comment]
:+1: otherwise.

File: moduleroot/.travis.yml (hunk 1 above)
Comment (mmoll) [inline moduleroot/.travis.yml]: indeed.

**Comment by domcleal** [comment]
Merged, thanks @mmoll!

**Pull request merged**